frontend/src/plugins/impl/SliderPlugin.tsx

```diff
@@ -54,16 +51,9 @@ const SliderComponent = (props: SliderProps): JSX.Element => {
     [setValue]
   );
   const id = useId();
-  const labelElement =
-    props.label === null ? null : (
-      <Label htmlFor={id}>
-        <HtmlOutput html={props.label} inline={true} />
-      </Label>
-    );
 
   return (
-    <div className={clsx(labelStyles.labelContainer, "mb-2")}>
```

Comment:
this bottom margin was removed too. i think its a bit more composable without an enforced margin, but lmk if this breaks some UI